Library/Formula/cv.rb

```diff
@@ -0,0 +1,15 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/BestPig/cv"
```

Comment:
We won't include this until we can have it pointing to the non-forked version, sorry!